pkg/containerd/events.go

```diff
@@ -192,9 +188,9 @@ func (e *EventMonitor) IsServing(ctx context.Context) error {
 
 // Close closes the client connection to the API server.
 func (e *EventMonitor) Close() error {
-	e.portTracker.RemoveAll()
+	defer e.containerdClient.Close()
 
-	return e.containerdClient.Close()
+	return e.portTracker.RemoveAll()
```

Comment:
```suggestion
	err := e.portTracker.RemoveAll()
	if err2 := e.containerdClient.Close(); err == nil && err2 != nil {
	  err = err2
	}
	return err
```
I assume the point is to return an error if either fails.